attack.go

```diff
@@ -107,6 +111,17 @@ func attack(opts *attackOpts) (err error) {
 		src = files[opts.targetsf]
 		hdr = opts.headers.Header
 	)
+	if opts.gzip {
+		buffer := new(bytes.Buffer)
+		w := gzip.NewWriter(buffer)
+
+		hdr.Set("Content-Type", "application/gzip")
```

Comment:
We should be setting the `Content-Encoding header`: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding